style.css

```diff
@@ -84,6 +85,150 @@ a {
   justify-content: space-between;
   align-items: center;
   margin-top: 20px;
+  margin-bottom: 40px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: 3;
+  gap: 142px;
+  align-items: center;
+  background-color: #f7f7f7;
+  justify-content: center;
+  padding-top: 100px;
+  padding-bottom: 100px;
+}
+
+.work {
+  padding: 24px;
+  border: gray solid 0.3px;
+  border-radius: 20px;
+  background: white;
+  display: flex;
+}
+
+.work-subtitle {
+  display: flex;
+  gap: 12px;
+  align-items: center;
+}
+
+.work-title {
+  font-weight: 600;
+  font-size: 3em;
+  color: black;
+}
+
+.work-details {
+  align-items: center;
+  padding-left: 46px;
+  padding-right: 46px;
+}
+
+.tags {
+  display: flex;
+  gap: 12px;
+  margin-bottom: 20px;
+}
+
+.tag {
+  color: #6170ff;
+  background-color: #ebebff;
+  padding: 10px;
+  border-radius: 10px;
+}
+
+.company {
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.work-link {
+  color: #6071fe;
+  text-decoration: none;
+  margin-top: 20px;
+  font-size: medium;
+  border: #6071fe solid 1px;
+  border-radius: 10px;
+  padding: 10px;
+}
+
```

Comment:
- [ ] As it is in the project requirements, you are expected to implement styles based on user interaction with the buttons on your page (for example: `'Hover'`, `"Active"`, ...) This is important because it improves user experience while interacting with website. Please implement them on these `See project` button and also the `Get my Resume` button

You may refer to the design guidelines given in the Figma design
![image](https://user-images.githubusercontent.com/90222110/185668139-7ddd7bec-937d-4f4c-979c-c9280104144a.png)